website/pages/blog/product-updates-5.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "CloudQuery Product Updates #4"
+tag: product
+date: 2023/04/09
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here!
+
+## New Support Pricing Page
+
+Check out our new [support pricing page](https://www.cloudquery.io/pricing)!
+
+## Destinations
+
+### New Destinations
+
+We've added 3 new destinations to CQ!
+
+* [DuckDB](https://www.cloudquery.io/docs/plugins/destinations/duckdb/overview)
+* [ClickHouse](https://www.cloudquery.io/docs/plugins/destinations/meilisearch/overview)
+* [Gremlin](https://www.cloudquery.io/docs/plugins/destinations/gremlin/overview)
+
+## Sources
+
+### New Sources
+
+We've added 5 new official source plugins to CQ around marketing,infra and databases!
+
+* [Google Ads](https://www.cloudquery.io/docs/plugins/sources/googleads/overview)
+* [Google Analytics](https://www.cloudquery.io/docs/plugins/sources/googleanalytics/overview)
+* [Facebook Marketing](https://www.cloudquery.io/docs/plugins/sources/facebookmarketing/overview)
+* [AWS Pricing](https://www.cloudquery.io/docs/plugins/sources/awspricing/overview)
+* [MySQL](https://www.cloudquery.io/docs/plugins/sources/mysql/overview)
+
+Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md).
```

Comment:
```suggestion
Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/6343cc6c7e699410e0629e44f4e3f7a0a4e3e809/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/6343cc6c7e699410e0629e44f4e3f7a0a4e3e809/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/6343cc6c7e699410e0629e44f4e3f7a0a4e3e809/plugins/source/azure/CHANGELOG.md).
```

This pins the change-log to the current head of `main`, so if we push new changes it won't change